store/src/java-test/com/zimbra/cs/service/account/GetInfoTest.java

```diff
@@ -0,0 +1,94 @@
+package com.zimbra.cs.service.account;
+
+import com.zimbra.cs.account.MockProvisioning;
+import com.zimbra.cs.account.Provisioning;
+import com.zimbra.cs.account.Server;
+import com.zimbra.cs.mailbox.MailboxTestUtil;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GetInfoTest {
+
+    private static Server server = null;
+    private String[] servicesEnabled = null;
+    private String[] servicesInstalled = null;
+    private String[] spellCheckUrls = null;
+
+    @BeforeClass
+    public static void init() throws Exception {
+        MailboxTestUtil.initServer();
+        MockProvisioning prov = new MockProvisioning();
+        Provisioning.setInstance(prov);
+        server = Provisioning.getInstance().getLocalServer();
+    }
+
+    private void setAttributes(String[] urls) {
+        Map<String, Object> map = new HashMap<>();
+        map.put("zimbraSpellCheckURL", urls);
+        server.setAttrs(map);
+    }
+
+    private void setAttributes(String[] servicesEnabled, String[] servicesInstalled, String[] urls) {
+        Map<String, Object> map = new HashMap<>();
+        map.put("zimbraSpellCheckURL", urls);
+        map.put("zimbraServiceEnabled", servicesEnabled);
+        map.put("zimbraServiceInstalled", servicesInstalled);
+        server.setAttrs(map);
+    }
+
+    /**
+     * spellcheck should return true when zimbraSpellCheckURL on the current server returns non-empty list of urls.
+     */
+    @Test
+    public void spellCheckIsTrueWhenItsEnabled() {
```

Comment:
I prefer not to use "Its" but describe a proper object name or such. Other functions as well.

Reply:
updated